src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = UserService.findByEmail(email);
+        if (user == null) {
+            return false;
+        } else if (password == user.getPassword()) {
+            return true;
+        }
         return false;
```

Comment:
let's simplify this code 
```suggestion
        return user != null && password.equals(user.getPassword());
```
also, remember it's better to use equals() if you're comparing String objects